chimedb/dataflag/client.py

```diff
@@ -338,14 +330,27 @@ def create_opinion(
     default=None,
     help="Type of flagging opinion to list. If not set, list all opinions.",
 )
-def opinion_list(type_):
+@click.option(
+    "--user",
+    "-u",
+    help="Wiki user to search. If no user is supplied, all users are searched.",
+    default=None,
+)
+@click.option("--revision", "-r", type=REVISION, default=None)
+def opinion_list(type_, user, revision):
     """List known revisions of TYPE."""
 
     query = orm.DataFlagOpinion.select()
 
     if type_:
         query = query.where(orm.DataFlagOpinion.type == type_)
 
+    if user:
+        query = query.where(db.mediawiki.MediaWikiUser.user_name == user)
```

Comment:
Wiki usernames
```suggestion
    if user:
        user = user[0].upper() + user[1:]
        query = query.where(db.mediawiki.MediaWikiUser.user_name == user)
```

Wiki usernames are weird.  They're stored with an initial capital letter, but the capitalisation of the first letter is ignored when doing username look-ups.

Reply:
Oh of course